---
output:
  md_document:
    variant: markdown_github
bibliography: refs_ITP.bib
csl: blood.csl
---

# **Platelet proteomics to understand the pathophysiology of immune thrombocytopenia: studies in mouse models**

Proteomics data analysis (DDA, label-free) performed in the following article:

Patricia Martínez-Botía, Marjolein Meinders, Iris M De Cuyper, Johannes A Eble, John W Semple, Laura Gutiérrez; Platelet Proteomics to Understand the Pathophysiology of Immune Thrombocytopenia: Studies in Mouse Models. *Blood Adv* 2022; bloodadvances.2021006438. doi: <https://doi.org/10.1182/bloodadvances.2021006438>

## Mass spectrometry data analysis

The RAW MS files were processed with the MaxQuant computational platform (version 1.6.17.0) running on Linux.[@cox2008maxquant; @sinitcyn2018maxquant] Searches were configured for label-free quantification (LFQ), with the options LFQ, match between runs and iBAQ, while the rest of the parameters were set to default. Peptides and proteins were identified using the Andromeda[@cox2011andromeda] search engine against the *Mus musculus* UniProt Swiss-Prot protein database (downloaded November 2020, 17,051 entries). The 'proteinGroups.txt' output table was further analyzed within the R environment (version 4.0.3) (R Project for Statistical Computing).[@rlanguage] Proteins were filtered for potential contaminants, only identified by site and reverse hits, and iBAQ intensities were log2-transformed. We did not further consider biological replicates that did not pass quality control tests (namely \> 35% of missing values). This resulted in the exclusion of some samples that would otherwise have confounded the results.

Protein hits were filtered in when they were present in all samples of at least one group. Intensities were quantile normalized, followed by imputation of missing data by the quantile regression imputation of left-censored data (QRILC) approach.[@imputeLCMD] Next, we removed the batch effect created by the different mouse models using the conservative 'removeBatchEffect' function from the 'limma' package.[@ritchie2015limma] The resulting dataset was subsequently fed to a linear model combined with empirical Bayesian statistics implemented by this same package, for the differential expression analysis of proteins among every pairwise comparison. Differential expression of protein hits with adjusted p-values (Benjamini-Hochberg method) lower than 0.05 were considered significant, in any given comparison, and were used for further analysis.

Data have been deposited to the ProteomeXchange Consortium via the PRIDE partner repository with the dataset identifier PXD028814.

Code for this part are the `prep_data.R` and `multiple_anova.R` scripts. Results from the first one are fed into the second script. Starting data is 'proteinGroups_mbr_v1.6.17.txt', which can also be found in ProteomXchange, along with the raw data.

## Correlation-based network analysis

Weighted gene correlation network analysis (WGCNA) was performed on the differentially expressed proteins (DEPs) using the 'WGCNA' R package[@langfelder2008wgcna] and the tailored workflow designed by Wu et al. (2020) for proteomics data.[@wu2020workflow] A 0.85 correlation coefficient threshold was chosen, while a soft threshold of 26 was selected for the construction of the weighted correlation and adjacency matrix by using the approximate scale-free network criteria. Of note, a signed network is constructed, so that only positively correlated proteins have a strong connection. Next, the topology overlap metrics (TOM) and its distances (1 -- TOM) are calculated from the adjacency matrix in a step-wise fashion. The resulting protein sets are hierarchically clustered (average method) based on the TOM distance, and an optimal set of modules was determined by using a dynamic tree-cutting algorithm, with a minimum of 10 proteins in each module, followed by merging close clusters (cut height of 0.1). Additionally, module eigenproteins (ME) are generated by value decomposition of the first principal component and signed module memberships (kME) are obtained. Finally, module-trait (Pearson) correlations were calculated.

Code for this part can be found in the `wgcna_analysis.R` script, and accompanying metadata are 'groups_wgcna.xlsx' and 'groups_traits.xlsx'. Data come from the output of the `multiple_anova.R script`.

## Annotation, gene ontology (GO) and pathway enrichment

UniProt IDs were mapped to their respective Entrez Gene IDs, Symbols and Gene names using the 'AnnotationDbi'[@AnnotationDbi] (version 1.52.0) and 'org.Mm.eg.db'[@orgMmegdb] (version 3.12.0) R packages. Gene Ontology (GO), Kyoto Encyclopedia of Genes and Genomes (KEGG), and Reactome enrichments on gene lists were obtained using the 'goseq'[@goseq] (version 1.36.0), and the 'msigdbr'[@MSigDB] (version 7.2.1) R packages. Intensity bias was taken into account, and the enrichment was calculated using the Wallenius approximation. The resulting P values were adjusted for multiple testing using the Benjamini-Hochberg correction, and an adjusted P value of \<0.05 was considered significant. Barplots were constructed with 'ggplot2'[@ggplot2], 'dplyr'[@dplyr], 'stringr'[@stringr] and 'wesanderson'[@wesanderson] R packages.

Code from this last part can be found in the `enrichment.R` and `vis_enrich.R` scripts. The former contains the enrichment analysis, while the latter contains the figures. For both of them, it is necessary to run the script(s) for every module/color.

# References
